templates/partial/_footer.html

```diff
@@ -1,3 +1,34 @@
+{% if is_docs %}
```

Comment:
Expanding footer partial to support 2 versions of it - one for standard site, another for full with docs layout.